extensions/arc/src/test/common/date.test.ts

```diff
@@ -0,0 +1,85 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as should from 'should';
+import 'mocha';
+import { fromNow } from '../../common/date';
```

Comment:
Thanks for the tests.  I'm thinking we should have symlinked date.ts instead of coping it.